libs/blog/shell/feature-shell-web/src/lib/root-shell.component.ts

```diff
@@ -34,11 +45,30 @@ import { SearchComponent } from '@angular-love/blog/search/feature-search';
     FooterComponent,
     LayoutComponent,
     SearchComponent,
+    AdBannerComponent,
+    NgClass,
   ],
 })
 export class RootShellComponent {
   readonly translocoService = inject(TranslocoService);
   readonly localizeRouterService = inject(LocalizeRouterService);
+  adBannerClosedClicked = signal(false);
+
+  adBannerVisible = computed(() => {
+    //TODO: after data-binding is done we can set this boolean if wp sends banner data
+    const adDataLoaded = true;
```

Comment:
i know it's on todo but maybe put it already as signal `adDataLoaded = singal(true) //TODO: ...`

Reply:
Done